Update principal_filter_analysis.ipynb

aviswaroop opened the pull request:
Thank you for your interest in contributing to DNIKit! Please review [CONTRIBUTING.md](../CONTRIBUTING.md) first. If you would like to proceed with making a pull request, please indicate your agreement to the terms outlined in CONTRIBUTING.md by checking the box below. If not, please go ahead and fork this repository and make your updates.

We appreciate your interest in the project!

Do not erase the below when submitting your pull request:

- [x] I agree to the terms outlined in CONTRIBUTING.md

---------------------------------------------

**One sentence summary of PR:**
 A few basic suggestions to improve the documentation's quality

## Checklist
( Replace '[ ]' with '[x]' if completed. )

- [x] I've searched through existing [Pull Requests](https://github.com/apple/dnikit/pulls) and can confirm my PR has not been previously submitted.
- [ ] I've written new tests for my core changes, as applicable.
- [ ] I've tested these changes locally (see [Contributor's Guide](https://apple.github.io/dnikit/dev/contributing.html)).
- [x] I've updated [CHANGELOG.md](https://github.com/apple/dnikit/blob/main/CHANGELOG.md) and added [documentation](https://github.com/apple/dnikit/tree/main/docs) for major new features.
- [x] My PR is of reasonable size for someone to review. (You may be asked to break it up into smaller pieces if it is not.)

***If you did not check one of these boxes, please explain why:***
 This PR does not contain any changes made to the code and instead changes were made only to the written documentation. Therefore it was not necessary to write new tests or test any of the changes locally.

## Description

Please provide enough information so that others can review your pull request.

### What existing problem does this PR aim to solve?

There are no issues pertaining to the code. Being a student my ability to grasp the concepts are limited. It is from this perspective, that I sought to suggest a few edits to the documentation for better understanding.

### New Behavior

Please find below a list of minor changes recommended to the documentation.

- Line 56: Replace '1a' with ' 1.a' to follow the notebook's prevalent format
- Line 113:
  - i) Replace 'Similarly' with 'Similar'
  - ii) The sentence, "This is the set of layers that to analyze using PFA", lacks clarity. This could be rephrased as "This is the set of layers that will be used to analyze with PFA". (Please note that my interpretation of the program could be wrong.)
  - iii) Replace '1.f' with '1.e'
- Line 149: Add 'that'
- Line 193: '1.f' has been used when there is no prior '1.e'. Hence, replace '1.f' with '1.e'
- Line 291: Add 'be' in between 'also' and 'queried'


### Type
Choose what is applicable, delete the others.
Bug fix (non-breaking change which fixes an issue)

### Priority
Choose high, medium, or low priority and why. (Ex: High priority, blocks X completely.)
Low priority, the documentation is already superb and this PR only seeks to make the documentation a little easier to read.

aviswaroop committed:
Update principal_filter_analysis.ipynb

Greetings,
Please find below a list of minor changes recommended to the documentation.

Line 56: Replace '1a' with ' 1.a' to follow the notebook's prevalent format

Line 113:
i) Replace 'Similarly' with 'Similar'
ii) The sentence, "This is the set of layers that to analyze using PFA", lacks clarity. This could be rephrased as "This is the set of layers that will be used to analyze with PFA". (Please note that my interpretation of the program could be wrong.)
iii) Replace '1.f' with '1.e'

Line 149: Add 'that'

Line 193: '1.f' has been used when there is no prior '1.e'. Hence, replace '1.f' with '1.e'

Line 291: Add 'be' in between 'also' and 'queried'

Thank you

mmaherwelsh commented on notebooks/model_introspection/principal_filter_analysis.ipynb:
```diff
@@ -110,7 +110,7 @@
    "source": [
     "### 1.c Find Convolution layers\n",
-    "Similarly to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that to analyze using PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.f).\n",
+    "Similar to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that will be used to analyze with PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.e).\n",
```
```suggestion
    "Similar to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that will be analyzed with PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.e).\n",
```

mmaherwelsh reviewed (changes requested):
Thanks for the contributions! There's one wording change I proposed,  but otherwise looks good to go.

aviswaroop committed:
Update notebooks/model_introspection/principal_filter_analysis.ipynb

Greetings,
Thank you very much for reviewing this PR. Your proposed wording change has helped me understand the code better.

Thank you

Co-authored-by: Megan Maher Welsh <3497799+mmaherwelsh@users.noreply.github.com>

mmaherwelsh reviewed (approved):
Looks great, please squash merge when you're ready!
